md/pages.md

```diff
@@ -95,13 +95,21 @@ out.write("""<iframe src=../portal.js/#/admin/monitoring/${idProcess}-${idCase}?
 
 While you are developing a custom page, you can enable custom page debug mode for your portal. In debug mode, you can see changes to your custom page without importing a new zip archive.
 
+::: info
+Before use setup tool for the studio context, you need to edit the file : `database.properties`, and re-define `h2.database.dir=../../<yourRepository>/h2_database`  
+:::
+
 To enable custom page debug mode, edit [`console-config.properties`](BonitaBPM_platform_setup.md) and set `custom.page.debug` to `true`.
 
 To work on a page in debug mode:
 
-1. Import your custom page zip archive into the portal. This creates a directory `<java.io.tmpdir>/bonita_portal_*/tenants/tenant_id/pages/<your custom page>`.
+::: info
+`<java.io.tmpdir>` for Tomcat platform is `<TOMCAT_HOME>/temp/`
```

Comment:
The way we generally phrase our sentences in the documentation whould rather be like:

On a Tomcat installation, `<java.io.tmpdir>` points to `<TOMCAT_HOME>/temp/`